src/calendar/CalendarMonth.jsx

```diff
@@ -32,6 +26,12 @@ const CalendarMonth = React.createClass({
     value: CustomPropTypes.momentOrMomentRange,
   },
 
+  componentWillMount() {
+    const lang = moment().localeData();
```

Comment:
@rolyatmax 
`require('moment/locale/${language}')` + `moment.locale(${language})` will change the locale of the global moment object, so whenever we use moment, it will have that locale set..

In our case we refresh the whole page as we need to load translations for messages etc.. so when bootstrapping the app we set the locale, before rendering the the component... 

I think the best solution should be merging #119 with #56.